spec/triggers/gcp-pub-sub.md

```diff
@@ -0,0 +1,18 @@
+#  GCP PubSub Trigger
+
+This specification describes the `gcp-pub-sub` trigger.
+
+```yaml
+triggers:
+- type: gcp-pubsub
+  metadata:
+    subscriptionSize: "5"
```

Comment:
No mention of required so this is probably optional? What is the default value?

You might want to add that here and in the description below?